src/presets/cloudflare/utils.ts

```diff
@@ -118,7 +119,7 @@ async function writeCFPagesHeaders(nitro: Nitro) {
     ([_, routeRules]) => routeRules.headers
   )) {
     const headers = [
-      path.replace("/**", "/*"),
+      withBase(path.replace("/**", "/*"), nitro.options.baseURL),
```

Comment:
`withBase` might cause one implicit issue:

Imagine if baseURL is `/test` and we have a route roule also for `/test/**` (So full URL would be `/test/test/**`) but in this case, we apply it to `/test/**` -- effectively `/**` without base)

Reply:
Yes but we can have another issue if the user set the baseURL to `/base/` and in the routeRules:

```typescript
baseURL: '/base/',
routeRules: {
  '/base/foo': { redirect: '/base/bar' }
}
```

But I guess we should advice to not include the base URL in the route rules so the user should do:

```typescript
baseURL: '/base/',
routeRules: {
  '/foo': { redirect: '/bar' }
}
```

Reply:
Yes indeed. baseURL is a dynamic config prefixing everything, including rules.